docs/frontend/frontend-development.md

```diff
@@ -1,20 +1,117 @@
-together, which means you can import styles from other bundles from within
-FrameworkCoreBundle. Bower components are also added to the import path, so you
-can easily import sass files from plugins when available. Try to use Bower as
-much as possible for frontend plugins to make upgrading and maintaining easier.
+components as possible through the variables file. This makes the code easier to maintain.
+
+### Overwrite variables
+Make a file `_variables.scss` in the folder styles. In this file you can set all bootstrap variables you want to override.
+Import this file in `style.scss` and in `style-dark.scss`.
+
+#### Differences in variables for dark mode
+Make an extra file `_variables-dark.scss` in the folder styles and import this file in `style-dark.scss`.
+
+### Custom components or extensions
+Create a folder `components` in the styles folder. Add your new component sass file in the new folder.
+To make it easy on yourself, make a `_components.scss` file in de styles folder. In this file you can import all your new components.
+Example: `@import 'components/component-name';`
+
+Now you can just easily import the `_conponents.scss` in your `style.scss` and `style-dark.scss`.
+So know in the future, you only need to add new component sass files in 1 file (`_components.scss`) instead of in 2 files.
+
+#### Differences in custom component for dark mode
+Make a extra folder, for example `components-dark`, and add your `_your-component.scss` file with the same name in this folder.
+
+You can again make a collector file for all dark components and import that file in `style-dark.scss`, 
+but in this scenario it's overkill.
+You can just import your custom dark mode component file in `style-dark.scss` after the import of `_components.scss`.
+
+Example: `@import 'components-dark/component-name';`
+
+
+### Overview folder structure
+```$xslt
+- assets
+    - styles
+        - components
+            - _component-name.scss
+        - _variables.scss
+        - mail.scss
+        - style.scss
+        - style-dark.scss
+```
+
+You can always fall back on the folder structure or existing components in [FrameworkStylepackage](https://github.com/sumocoders/FrameworkStylePackage/tree/master/src/sass).
+
+## JS
+
+You can find `app.js` in the folder `assets/`. This is the main collector for the Javascript.
+The file `imports.js` is imported in `app.js`.
+These imports are the default components from our [FrameworkStylepackage](https://github.com/sumocoders/frameworkStylePackage).
+
+We use ES6 classes for each Javascript component. More info [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes)
+
+### Custom components or extensions
+Create a folder `components` in the js folder. Add your new component js file in the new folder.
+Build you ES6 class in a new file with as name the name of your component. Do not forgot to import you new component file in `app.js`.
```

Comment:
Build you ES6 class in a new file with as name the name of your component → 
Build your ES6 class in a new file that has the same name as your component.

Reply:
fixed in ce0d873